### ISSUE TITLE
Questions about GCP X OpenTelemtry

### ISSUE THREAD

**Issue by wagnerww** [GoogleCloudPlatform/opentelemetry-operations-java#201]
If I opt for auto-instrumentation, do I still need this google dependency in case I need to generate a span or metric?

I'm having trouble generating the trace, and I don't have an example that works:
wARN io.opentelemetry.api.GlobalOpenTelemetry - You are currently using the OpenTelemetry Instrumentation Java Agent; all GlobalOpenTelemetry.set calls are ignored - the agent provides the global OpenTelemetry object used by your application.
java.lang.Throwable
         at io.pentelemetry.api.GlobalOpenTelemetry.set(GlobalOpenTelemetry.java:97)

**Comment by dashpole** [comment]
You should be able to follow the setup guide in https://github.com/GoogleCloudPlatform/opentelemetry-operations-java/tree/main/exporters/auto#auto-configuration-for-opentelemetry-in-google-cloud to use auto-instrumentation with the GCP exporter.  Let me know if you have further questions.

**Comment by psx95** [comment]
@wagnerww did the above comments answered your question ?

**Comment by psx95** [comment]
@wagnerww We currently do not have any example showcasing Anthos with OpenTelemetry, but the OTel auto-instrumentation should "just work" with Anthos Service Mesh - without any extra configuration. 

If you set up context propagation in your apps, you will get end-to-end traces consisting of mesh spans.
If you set up trace export in your apps in addition to propagation, you will get additional spans with details from your app.

**Comment by wagnerww** [comment]
@psx95 In this case, would I have to apply the agent and the exporter in the docker image?

Another question....for the exporter and the opentelemetry agent, what is the minimum java version?

I did some tests with the image "openjdk:17-alpine", without success. But with "openjdk:17" it worked.

**Comment by psx95** [comment]
@wagnerww Yes, I think you would add both JARs (auto-exporter and the agent) as JVM arguments to the docker container and it should start giving you telemetry data. You can refer to this [configuration](https://github.com/GoogleCloudPlatform/opentelemetry-operations-java/blob/main/examples/autoinstrument/build.gradle#L50). This configuration contains jvmFlags that configure both the JARs.  


All of our examples use java 11, but they should work with Java 17 too. I think it maybe because Alpine is a lightweight image so it may be missing a few libraries required to run our exporters. 



*We recently increased the gradle version in [this PR](https://github.com/GoogleCloudPlatform/opentelemetry-operations-java/pull/209) for this repo to be compatible with Java 17. So we may upgrade the examples to use 17 in future.*
